src/dev_config.yaml

```diff
@@ -76,7 +76,7 @@ network_paths:
   frozen_data_staged_output_path: "02_freezing/frozen_data_staged/"
   frozen_data_staged_path: "02_freezing/frozen_data_staged/2023_FROZEN_staged_BERD_full_responses_24-08-08_v892.csv"
   freezing_changes_to_review_path: "02_freezing/changes_to_review/"
-  freezing_additions_path: "02_freezing/freezing_updates/"
+  freezing_additions_path: "R:/BERD Results System Development 2023/DAP_emulation/2023_surveys/BERD/02_freezing/changes_to_review/2023_freezing_additions_to_review_24-08-09_v934.csv"
```

Comment:
Additions and amends you can test with:
  freezing_additions_path: "02_freezing/freezing_updates/JC_2023_freezing_additions_to_review_24-08-09_v925.csv"
  freezing_amendments_path: "02_freezing/freezing_updates/JC_2023_freezing_amendments_to_review_24-08-09_v925.csv"